### DISCUSSION TITLE
LocalSend has low speed.

### DISCUSSION THREAD

**Discussion by roman90sv** [localsend/localsend#1990] [Discussion (English, International)] [answered]
I compared it with Warpinator and HTTP-Server in MixPlorer. I sent a 2GB file from my PC (Linux Debian, SSD drive, wifi6-router) to a tablet (Huawei Matepad 11.5 2023). The speed fluctuated between 90 and 150 megabits. In Warpinator the speed was remained at 700 megabits. When sending a file through the Firefox browser via the HTTP protocol (to MixPlorer application on my tablet), the speed was more than 950 megabits and was very stable, without any drops. Warpinator application for Android is not even official (it is a port from third-party developers). And it works faster.

Why is LS so slow?

**Comment by Tienisto** [comment, marked as answer]
Try disable encryption. Also, do you use the latest version?

**Reply by roman90sv** [reply]
> Try disable encryption. Also, do you use the latest version?

Speed became equal to the warpinator without encryption.
ver. 1.15.4, both on Linux and Android.

But mixplorer is still the champion.
